eu/ja-jp/content/topics/end-user/ov-mng-account-windows.htm

```diff
@@ -265,27 +265,24 @@
-                                                        </li>
-                                                        <li><a href="ov-remove-account-windows.htm" class="MCXref xref">WindowsデバイスのOkta Verifyからアカウントを削除する</a>
-                                                        </li>
-                                                    </ul>
-                                                    <h2>関連トピック</h2>
+                                                    <h2><span class="mc-variable okta-variables.ProductName variable">Okta</span>で保護されたアプリにアクセスしてアカウントを追加する</h2>
+                                                    <p><span class="mc-variable okta-variables.ProductName variable">Okta</span>の保護対象アプリにWindowsデバイスからアクセスし、<span class="uicontrol">［<span class="mc-variable okta-feature-names.Okta_FastPass variable">Sign in with Okta FastPass（Okta FastPassでサインインする）］</span></span>を選択すると、<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>はアカウントの有無を確認します。まだアカウントを持っていないときは、アプリのセットアップウィザードが示す手順に従います。  </p>
+                                                    <h2>Windows Helloを構成する</h2>
+                                                    <p><span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>を開き、自分のアカウントをクリックします。トグルをクリックして<span class="uicontrol">Windows Hello confirmation（Windows Hello確認）</span>のオン/オフを切り替え、手順に従います。</p>
+                                                    <p>組織でWindows Helloの使用が義務付けられている場合、<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>はこれをオンにするように促します。 </p>
+                                                    <p><span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>のWindows Helloとデバイス設定が同期していないときは、メッセージが表示されたら<span class="uicontrol">［Sync（同期）］</span>をクリックします。</p>
+                                                    <h2><span class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>認証のデフォルトアカウントを設定する</h2>
+                                                    <p>複数の<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>アカウントがあるときは、最も使用頻度の高いアカウントを<span class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>を使ったサインインのデフォルトアカウントとして設定できます。デフォルトアカウントは変更可能です。</p>
+                                                    <p><span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>を開き、アカウントをクリックします。［<span class="uicontrol">Set as default for </span><span style="font-weight: bold;" class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>（Okta FastPassのデフォルトに設定する）］リンクをクリックします。<span class="wintitle">［Account Details（アカウントの詳細）］</span>ページの右上端に<span class="uicontrol">［</span><span style="font-weight: bold;" class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>のデフォルト］インジケーターが表示される場合、選択したアカウントはすでにデフォルトとして設定されています。 </p>
```

Comment:
Something odd is happening here. The `<span>` elem for "Okta FastPass" is not formatted correctly, need to check EN to see where source of this error is